config/v201/core_migrations/1_up-initial.sql

```diff
@@ -86,3 +86,9 @@ CREATE TABLE METER_VALUE_ITEMS (
     ENCODING_METHOD TEXT,
     PUBLIC_KEY TEXT
 );
+
+CREATE TABLE CHARGING_PROFILES (
+    ID INT PRIMARY KEY NOT NULL,
+    EVSE_ID INT NOT NULL,
```

Comment:
Looking at the get and clear charging profile functionality it might be beneficial to also store the stacklevel and chargingProfilePurpose as separate values in the database, next to the json. That would allow us to get and delete directly on the database layer for most of those calls.

Reply:
I like this for a long term solution. Could this be part of this [issue](https://github.com/EVerest/libocpp/issues/713)?

Reply:
If possible I would prefer for you to add it to this PR already, even if you don't really implement this yet. If we don't do it in this PR we need to add an additional migration file for adding these fields. I can give you a proposal for this if that would help?

Reply:
I can get the update done. Do you want the crud portions in the `DatabaseHandler` or just the table updates?

Reply:
The crud portions can be part of the clear and get charging profile use cases I think so just the table updates should be fine.

Reply:
This is done.